dss-api.yml

```diff
@@ -1383,9 +1383,16 @@ paths:
                       type: string
                       format: DSS_VERSION
                     name:
-                      description: Name of the file.
+                      description: >
+                        Name of the file.
+
+                        - Absolute paths (beginning with a slash) are not allowed.
+                        - Relative paths must not have unix shortcuts (i.e. ".", "..", "~/")
+                        - No "\" (back)slashes allowed.  e.g. "C:\\path" or "path\file.json"
+                        - Only relative paths with "/" (forward)slashes allowed.  e.g. "path/file.json" or "file.json"
+                        - Please upload using "/" (forward)slashes, even if on Windows.  DL will convert back.
```

Comment:
expand DL to downloads or remove last sentence.

Reply:
The API will not modify the slashes in get file or bundle. This would be done in the CLI.

Reply:
I'll remove the last sentence.  I realize it was discussed offline and had planned to make the corresponding changes in `dcp-cli`, but that that should be a separate PR.  I'll create an issue for it.